app/models/timecard.rb

```diff
@@ -10,27 +10,11 @@ class Timecard < ActiveRecord::Base
   INTENTION_CHOICES = ['Unknown', 'Available', 'Scheduled','Not Needed']
   OUTCOME_CHOICES = ["Not Needed", 'Worked', 'Unavailable', 'AWOL', 'Vacation']
 
-  def intended_end_date_cannot_be_before_intended_start
-    if ((!intended_end_time.blank?) and (!intended_start_time.blank?)) and intended_end_time < intended_start_time
-      errors.add(:intended_end_time, "must be after the start, unless you are the Doctor")
-    end
-  end
-
-  def actual_end_date_cannot_be_before_actual_start
-    if ((!actual_end_time.blank?) and (!actual_start_time.blank?)) and actual_end_time < actual_start_time
-      errors.add :actual_end_time, "must be after the start, unless you are the Doctor"
-    end
-  end
+  validates_presence_of :person_id, :event_id
```

Comment:
A future PR will need to remove the link to events. A Timecard represents a person working for a period of time, which could include multiple events.